test/double_spend_test.go

```diff
@@ -44,12 +44,14 @@ func TestDoubleSpend(t *testing.T) {
 
 			// send double spending transaction when first tx is in mempool
 			txMempool := createTxToNewAddress(t, privateKey, utxos[0])
-			postTxChecksStatus(t, arcClient, txMempool, Status_REJECTED, tc.extFormat)
+			postTxChecksStatus(t, arcClient, txMempool, Status_DOUBLE_SPEND_ATTEMPTED, tc.extFormat)
```

Comment:
What would be really nice here is if it could be checked whether the double spend attempting txs can be found in the response